jerry-core/ecma/operations/ecma-typedarray-object.c

```diff
@@ -897,6 +898,11 @@ ecma_op_typedarray_set_index_prop (ecma_object_t *obj_p, /**< a TypedArray objec
     return true;
   }
 
+  if (ECMA_IS_VALUE_ERROR (error))
```

Comment:
What about merging the two if statements like this?

```c
if (ECMA_IS_VALUE_ERROR (error))
{
  ecma_free_value (JERRY_CONTEXT (error_value));
  return false;
}

JERRY_ASSERT (ecma_is_value_empty (error));
return true;
```

Reply:
done